src/containers/WelcomePage/WelcomePage.tsx

```diff
@@ -72,6 +76,16 @@ interface Props {
 const WelcomePage = ({ locale, skipToContentId }: Props) => {
   const { t } = useTranslation();
 
+  useEffect(() => {
+    getData();
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
+
+  const [fetchData, { data }] = useLazyQuery<GQLAlertsQuery>(alertsQuery);
+
+  const getData = () => {
+    fetchData();
+  };
+
```

Comment:
Er det noen grunn til at vi gjør dette fremfor å bruke en vanlig query?

Reply:
Eg tenkte det var lurt med lazy for å ikkje blokkere lasting av forsida.

Reply:
Lurt å ikke blokkere lasting, ja! Tror dog vi kunne oppnådd samme effekt ved å ikke basere oss på loading-staten til query. Queryen kan enten plasseres i toppen av `WelcomePage` eller flyttes inn i en egen `UptimeAlerts`-komponent. Skal ikke blokkere denne av den grunn, derimot.

Reply:
Kom gjerne med et forslag til en bedre visning.